src/client/connection.rs

```diff
@@ -508,7 +429,7 @@ impl AsyncRead for Connection {
     }
 }
 
-impl SqlReadBytes for Connection {
+impl<S: futures::AsyncRead + futures::AsyncWrite + Unpin + Send> SqlReadBytes for Connection<S> {
```

Comment:
You might want to use futures::{AsyncRead, AsyncWrite} to save valuable horizontal space.